client/src/components/Workflow/List/useWorkflowActions.ts

```diff
@@ -0,0 +1,110 @@
+import { computed, type Ref, ref } from "vue";
+
+import {
+    copyWorkflow as copyWorkflowService,
+    deleteWorkflow as deleteWorkflowService,
+    updateWorkflow as updateWorkflowService,
+} from "@/components/Workflow/workflows.services";
+import { useConfig } from "@/composables/config";
+import { useConfirmDialog } from "@/composables/confirmDialog";
+import { useToast } from "@/composables/toast";
+import { copy } from "@/utils/clipboard";
+import { withPrefix } from "@/utils/redirect";
+import { getFullAppUrl } from "@/utils/utils";
+
+// TODO: replace me with a more accurate type
+type Workflow = any;
```

Comment:
I'm not sure if the model you are looking for is already in:

https://github.com/galaxyproject/galaxy/blob/ff9470e1fc3d133416d2851fcdd36a87472b082f/client/src/api/workflows.ts#L3

But if it is not, it would be a good idea to add the type definition there and import the same definitions around (workflows.services.ts, WorkflowIndicators.vue, etc.) so when we have the proper model we just replace it in one place.

Reply:
@davelopez Thanks for pointing it out! I addressed it in 9a22b0e853c64c07b37194fbc97fd8a8453eb2a8.